src/devtools/client/debugger/src/components/Editor/EmptyLines.tsx

```diff
@@ -33,15 +33,21 @@ interface ELProps {
 type FinalELProps = PropsFromRedux & ELProps;
 
 class EmptyLines extends Component<FinalELProps> {
+  _rafId: number | null = null;
```

Comment:
It took me a minute to realize `raf` was `requestAnimationFrame`. I'm not saying we should spell out `requestAnimationFrame`, because that is long, but thought I'd mention it in case you or anyone else has a better suggestion.